src/landing/home-stats/home-stats-table.jsx

```diff
@@ -163,42 +165,56 @@ function getGridRowsAndColumns(stats) {
     }
   }
 
-  if (stats.topLists.blocked?.length) {
-    for (const blocked of stats.topLists.blocked) {
+  if (stats.topLists.blocked && Object.keys(stats.topLists.blocked).length) {
+    for (const [key, value] of Object.entries(stats.topLists.blocked)) {
       blockedData['topBlocked'].push({
-        ...blocked,
-        block_count: blocked.block_count.toLocaleString(),
+        did: key,
+        profile_url: null,
+        handle: value.handle,
```

Comment:
since we're not resolving anything ahead of time, not even the handle from the api, these properties should probably be removed entirely